### PR TITLE
Feature/fetch data

### PR DESCRIPTION
**Pull request by PaigeVannelli**
## Pull Request

<br>

### Description

* Is this a feature or a fix?
Feature - adding a new file to do our initial data fetch requests 

* Where should the reviewer start?
Review the APICalls.js file and the start app function in scripts

* What functionalities do these changes effect? Why were these changes made? What was the motivation behind these changes?
Will allow us to get data from the server rather than our data files in the project 

* What outside features or research did you use to implement this fix?
* n/a

* How should this be tested?
* n/a

<br>

***

#### Please review considerations below:

- [ ] Follows Turing Style Guidelines

- [ ] Changes generate no new warnings or errors

- [ ] Checked code for and corrected any spelling errors

- [ ] README has been updated(if applicable)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,21 +17,17 @@
   "license": "MIT",
   "devDependencies": {
     "chai": "^4.2.0",
-    "chai-spies": "^1.0.0",
```
Comment (percworld) [inline package.json:20]: I suppose you found a good reason we will not need chai spy

File: src/scripts.js [hunk 2]
```diff
@@ -14,6 +14,7 @@ import Activity from './Activity';
 import Hydration from './Hydration';
 import Sleep from './Sleep';
 import UserRepo from './User-repo';
+import fetchData from './APICalls';
```
Comment (percworld) [inline src/scripts.js:17]: I like your idea of calling all userData updated with helper functions.

**Review by percworld — APPROVED** [review]

File: src/APICalls.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    .catch(err => console.log(err.message))
+}
+
+const fetchHydrationData = () => {
+    fetch("http://localhost:3001/api/v1/hydration")
+    .then(response => response.json())
+    .then(hydrationData => console.log(hydrationData))
+    // function that would change dom display
+    .catch(err => console.log(err.message))
+}
```
Comment (cbmackintosh) [inline src/APICalls.js:23]: Are we sure this is the right place for DOM manipulation? Or should we keep DOM manipulation to scripts.js? Just reviewing David's solution to today's practice problems - it looks like he's keeping DOM manipulation separate from his API calls.
https://github.com/turingschool-examples/fe2-fetch-practice/blob/possible-solution/client/index.js

**Review by cbmackintosh — APPROVED** [review]
Nice work team. We'll likely need more API calls in the future, but these should help us get started with streamlining and consolidating the code base.

**Pull request merged**